style/battle.css

```diff
@@ -253,93 +281,126 @@ License: GPLv2
 	border: 0;
 	border-radius: 3px;
 }
+
 .statbar span.brn {
 	background: #EE5533;
 	color: #FFFFFF;
 }
+
 .statbar span.psn {
 	background: #A4009A;
 	color: #FFFFFF;
 }
+
 .statbar span.par {
 	background: #9AA400;
 	color: #FFFFFF;
 }
+
 .statbar span.slp {
 	background: #AA77AA;
 	color: #FFFFFF;
 }
+
 .statbar span.frz {
 	background: #009AA4;
 	color: #FFFFFF;
 }
+
 .statbar span.bad {
 	background: #FFE5E0;
 	color: #FF4400;
 	border-color: #FF4400;
 }
+
 .statbar span.good {
 	background: #E5FFE0;
 	color: #33AA00;
 	border-color: #33AA00;
 }
+
 .statbar span.neutral {
 	background: #F0F0F0;
 	color: #555555;
 	border-color: #555555;
 }
+
 .leftbar {
 	position: absolute;
 	top: 0;
 	left: 0;
 	height: 450px;
 	width: 100px;
 	background-color: #CCCCCC;
-	background-color: rgba(255,255,255,0.5);
-	border-right: 1px solid rgba(215,215,215,0.5);
+	background-color: rgba(255, 255, 255, 0.5);
+	border-right: 1px solid rgba(215, 215, 215, 0.5);
 	color: #777777;
 }
+
 .rightbar {
 	position: absolute;
 	top: 0;
 	right: 0;
 	height: 450px;
 	width: 100px;
 	background-color: #CCCCCC;
-	background-color: rgba(255,255,255,0.5);
-	border-left: 1px solid rgba(215,215,215,0.5);
+	background-color: rgba(255, 255, 255, 0.5);
+	border-left: 1px solid rgba(215, 215, 215, 0.5);
 	color: #777777;
 }
+
 .trainer {
 	text-align: center;
 	position: absolute;
 	left: 0;
 	right: 0;
-	bottom: 102px;
+	bottom: 75px;
 }
+
 .rightbar .trainer {
 	bottom: 232px;
 }
+
+#p1 {
```

Comment:
You can't! People might have multiple battles! This should be something like `.trainer.p1`.